cranelift-codegen/src/binemit/memorysink.rs

```diff
@@ -30,12 +30,14 @@ use core::ptr::write_unaligned;
 /// Note that `MemoryCodeSink` writes multi-byte values in the native byte order of the host. This
 /// is not the right thing to do for cross compilation.
 pub struct MemoryCodeSink<'a> {
+    // Pointer to start of sink's preallocated memory.
```

Comment:
nit: use `///` instead of `//` so this shows up in IDE help

Reply:
Done.